stored_queries/djornl/djornl_fetch_genes.yaml

```diff
@@ -0,0 +1,15 @@
+# Fetch a gene document by list of keys
+name: djornl_fetch_genes
+params:
+  type: object
+  required: [keys]
+  properties:
+    keys:
+      type: array
+      items: {type: string}
+      title: Gene Keys
+      examples: [[AT1G01010]]
+query: |
+  FOR g IN djornl_gene
+    FILTER g._key IN @keys
+    RETURN g
```

Comment:
Can you add queries to fetch node data, filtering on a search term (or terms) that appears (a) in a specified field, or (b) in any field?

Reply:
Arango has some good features for indexing a bunch of different fields to provide one big search endpoint ("ArangoSearch Views"). Will add a view for genes